app/controllers/application_controller.rb

```diff
@@ -275,11 +275,12 @@ def preferred_editor
   end
 
   def preferred_color_scheme(subject)
-    if current_user
-      current_user.preferences.find_by(:k => "#{subject}.color_scheme")&.v || "auto"
-    else
-      "auto"
-    end
+    scheme = current_user ? current_user.preferences.find_by(:k => "#{subject}.color_scheme")&.v : nil
```

Comment:
Can this not use safe navigation like this?
```suggestion
    scheme = current_user&.preferences.find_by(:k => "#{subject}.color_scheme")&.v
```

Reply:
Only with offending RuboCop. But not going into the subjects loop seems even better.